rpi5.yaml

```diff
@@ -0,0 +1,260 @@
+        - "../../meta-virtualization"
+        - "../../meta-raspberrypi"
+        - "../../meta-selinux"
+        - "../../meta-openembedded/meta-oe"
+        - "../../meta-openembedded/meta-networking"
+        - "../../meta-openembedded/meta-perl"
+        - "../../meta-openembedded/meta-python"
+        - "../../meta-openembedded/meta-filesystems"
+        - "../../meta-xt-common/meta-xt-domx"
+        - "../../meta-xt-common/meta-xt-driver-domain"
+        - "../../meta-xt-rpi5"
+
+      target_images:
+        - "tmp/deploy/images/%{RPI_MACHINE}/xen-%{RPI_MACHINE}"
+        - "tmp/deploy/images/%{RPI_MACHINE}/xenpolicy-%{RPI_MACHINE}"
+        - "tmp/deploy/images/%{RPI_MACHINE}/%{XT_XEN_DTB_NAME}"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bcm2712-rpi-5-b.dtb"
+        - "tmp/deploy/images/%{RPI_MACHINE}/%{XT_DOMD_IMAGE}-%{RPI_MACHINE}.rootfs.ext3"
+        - "tmp/deploy/images/%{RPI_MACHINE}/Image-%{RPI_MACHINE}.bin"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/bootcode.bin"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/config.txt"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/cmdline.txt"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/fixup.dat"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/fixup_cd.dat"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/fixup_db.dat"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/fixup_x.dat"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/fixup4.dat"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/fixup4cd.dat"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/fixup4db.dat"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/fixup4x.dat"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/start.elf"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/start_cd.elf"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/start_db.elf"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/start_x.elf"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/start4.elf"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/start4db.elf"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/start4cd.elf"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/start4x.elf"
+#      additional_deps:
+#        - "../%{ZEPHYR_DOM0_DIR}/%{ZEPHYR_DOM0_BUILD_DIR}/zephyr/zephyr.bin"
+
+  dom0:
+    default: true
+    build-dir: "%{ZEPHYR_DOM0_DIR}"
+    sources:
+      - *ZEPHYR_SOURCES
+
+    builder:
+      type: zephyr
+      board: "%{ZEPHYR_DOM0_MACHINE}"
+      target: "%{ZEPHYR_DOM0_TARGET}"
+      work_dir: "%{ZEPHYR_DOM0_BUILD_DIR}"
+      vars:
+        - "CONFIG_DOMD_BIN_PATH=\"../%{ZEPHYR_DOMD_BUILD_DIR}/zephyr/zephyr.bin\""
+        - "CONFIG_DOMD_DTB_PATH=\"../../%{YOCTOS_WORK_DIR}/%{DOMD_BUILD_DIR}/tmp/deploy/images/%{RPI_MACHINE}/bcm2712-rpi-5-b.dtb\""
+        - "CONFIG_DOMU_BIN_PATH=\"../%{ZEPHYR_DOMU_BUILD_DIR}/zephyr/zephyr.bin\""
+        - "CONFIG_XSTAT=y"
+        - "CONFIG_XSTAT_SHELL_CMDS=y"
```

Comment:
Is this required inside yaml? Maybe this has to go into some config files?

Reply:
Ok

Reply:
Deleted